src/components/Footer/Footer.tsx

```diff
@@ -1,7 +1,11 @@
+import FooterWrapper from "./styles";
+
 const Footer = () => {
   return (
-    <footer>HRM React Application</footer>
-  )
-}
+    <FooterWrapper className="flex">
```

Comment:
Remove class name  from styled-components and handle it in styled-components itself

Reply:
Correction done in Phase-1 branch.